source/storage.rst

```diff
@@ -0,0 +1,159 @@
+(from any VM running at any EGI provider or even from other cloud providers
+or from your own laptop/browser), you can expose the data via external portals
+(using HTTP as transport protocols), you can set access control lists per
+container and even make the data publicly available. On the other hand, 
+data is accessed via a API requests, thus integration with existing applications
+may require a change to the application logic.
+
+A summary of the main differences between Block and Object Storage is reported
+in the following table.
+
++--------------------+----------------------------------------+--------------+----------------------+---------------------------------+
+|                    | Access                                 | Sharing      | Accounting           | Usage                           |
++====================+========================================+==============+======================+=================================+
+| **Block Storage**  | only from within a VM only at the same | Not possible | for the entire block | POSIX access, use as local disk |
+|                    | site the VM is located                 |              |                      |                                 |
++--------------------+----------------------------------------+--------------+----------------------+---------------------------------+
+| **Object Storage** | from any device connected to the       | Possible     | only for the data    | via HTTP requests to server     |
+|                    | internet                               |              | stored               |                                 |
++--------------------+----------------------------------------+--------------+----------------------+---------------------------------+
+
+According to your application needs, you may select one technology over the
+other. In general, block storage is a good and simple solution for temporary
+data and data which you do not need to share beside the single application
+running on a single VM. If you need to have your data exposed within portals
+or shared between different steps of your processing workflow, it is usually
+best to use the object storage.
+
+
+Block Storage
+-------------
+
+Block storage is created and managed via requests to specific APIs. Once the
+storage is attached to a VM, is managed as a regular block device that can be
+managed from within the VM as any other block device.
+
+Management
+::::::::::
+
+OpenStack providers offer block storage using their native API. Both the 
+CLI client or dashboard can be used. Main commands for managing volumes are
+listed below:
+
++------------------------------------------+--------------------------------------+
+| Command                                  | function                             |
++==========================================+======================================+
+| `volume create --size <size> <name>`     | create a volume of size `<size>` GBs |
```

Comment:
You may want to use double backquotes here

Reply:
I do...